src/SFA.DAS.Campaign.Web/ViewComponents/TrainingProvider-Heading/TrainingProviderHeadingViewComponent.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Threading.Tasks;
```

Comment:
This folder naming is a little out of the norm.

Reply:
there are other examples already in the project of the same naming convention so not sure what the alternative would be?

Reply:
What's wrong with TrainginProviderHeading ?